Epic 2 bpmn 2.0

LinkKF opened the pull request.

LinkKF committed:
added Camunda Model API

LinkKF committed:
chain Tasks with Camunda and create BPMN 2.0 model

JP-1999 committed:
Maven Dependencies hinzugefügt
Erstes BPMN Modell mittels Actions und der Camunda API erstellt.

JP-1999 committed:
Merge branch 'Epic-2-BPMN-2.0' of https://github.com/woped/T2P into Epic-2-BPMN-2.0

JP-1999 committed:
Methode ausgelagert in die BPMN Exporter Klasse
Check für Dummy Node eingebaut
Object zu jeder Action hinzugefügt
Überlegungen für die Implementierung von Gateways

LinkKF committed:
return BPMN File to Frontend

LinkKF committed:
first adaptations for BPMN 2.0 in frontend

LinkKF committed:
added endEvent and fixed action names

LinkKF committed:
fixed errors

JP-1999 committed:
Zweite Implementierungsversuch von Gateways weg von Iterativen Methoden hin zu rekrusiver Methodik. In diesem Zustand Fehler im DOM.

JP-1999 committed:
Merge branch 'Epic-2-BPMN-2.0' of https://github.com/woped/T2P into Epic-2-BPMN-2.0

JP-1999 committed:
DOM Fehler behoben. Erste funktionierende Version mit BPMN Gateways.

DOM Fehler behoben. Erste funktionierende Version mit BPMN Gateways.

LinkKF committed:
Added comments and renamed method "goThrough"

LinkKF committed:
added endEvent to process

LinkKF committed:
Updated method name

LinkKF committed:
mapped BPMN 2.0 to new endpoint and recreated old endpoint

LinkKF committed:
corrected mapping for parallel gateways

NilsLeger commented on pom.xml:
```diff
@@ -241,14 +241,16 @@
+			<version>7.10.0</version>
 		</dependency>
 		<dependency>
-			<groupId>io.micrometer</groupId>
```
Wieso wurde hier einfach die Micrometer Dependency entfernt?

KanBen86 replied:
Die Dependency ist doppelt vorhanden. Siehe Zeilen 238-242

KanBen86 commented on pom.xml:
```diff
@@ -241,14 +241,16 @@
-			<scope>runtime</scope>
+			<groupId>org.camunda.bpm.model</groupId>
+			<artifactId>camunda-bpmn-model</artifactId>
+			<version>7.10.0</version>
```
Aber diese Dependency ist immernoch doppelt vorhanden.

Commit:
Reorganized dependencies

Commit:
Reorganized dependencies

KanBen86 reviewed (approved):
Im workspace quergeprüft, da die Ausgabe hier etwas unübersichtilich ist.

Commit:
Further fixes

removed empty test clazz,
update microversion in pom,
Dockerfil modifications,
openjdk base image update

Commit:
Changed Container Port to default 80

nileger committed:
refactor: Improve Dockerfile
